.gitignore

```diff
@@ -0,0 +1,3 @@
+*.md
+!README.md
+/node.module

```

Comment:
- [ ] Please your .gitignore should contain these 👇️
```
test.md
node_module
```
Copy them and paste them into `.gitignore` file.